### PR TITLE
Release 3.0.1

### LINKED ISSUE #2: scons command does not work on Windows
On Windows the `scons` command fails:

```
C:\User\builder>conda create -n foo python=3.6 scons -c conda-forge
...
C:\User\builder>activate foo

(foo) C:\User\builder>scons -h
Fatal Python error: Py_Initialize: unable to load the file system codec
ModuleNotFoundError: No module named 'encodings'
...
```

Running the scons.py Python script directly works fine:

```
(foo) C:\User\builder>python C:\Users\builder\Anaconda3\envs\foo\Scripts\scons.py -h
usage: scons [OPTION] [TARGET] ...
```

### PR DESCRIPTION
**Pull request by nicoddemus**
This also fixes #2 for me on Python 3.6 (on 3.5 I was not having this problem).

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -14,6 +14,7 @@ source:
 build:
   number: 0
   script: python setup.py install --standard-lib
+  noarch: python
```
Comment (nicoddemus) [inline recipe/meta.yaml]: With this #2 is fixed.

**Comment by nicoddemus** [comment]
@conda-forge-admin, please rerender

**Comment by nicoddemus** [comment]
@conda-forge-admin, please rerender

**Comment by nicoddemus** [comment]
@stuertz I believe this is now ready for merging. 👍

**Comment by ocefpaf** [comment]
Oops... Got here before @stuertz :grimacing:

**Comment by nicoddemus** [comment]
Thanks @ocefpaf!

**Pull request merged**